scripts/run_multiuser_tests.py

```diff
@@ -214,6 +214,8 @@ def test_global_wf_from_wdl():
         ))
         raise
 
+
+"""Commented out in APPS-1197 because of the fund limits on Bob's account
```

Comment:
or maybe just remove the test from integration and add separate one?
https://github.com/dnanexus/dxCompiler/blob/ed246aacb53bc955b7506f0718c377f0cff1abe7/.github/workflows/integration.yml#L67-L73